app/views/chambers/peerage/candidates/show.html.erb

```diff
@@ -67,6 +62,23 @@
     <%= f.button class: 'btn btn-default' %>
   <% end %> 
   </div>
+  </div>
+  <div role="tabpanel" class="tab-pane active " id="gallery">
+  <% if @candidate.documents %>
+    <%= render 'chambers/peerage/candidates/gallery', documents: @candidate.documents %>
```

Comment:
Consider always rendering the the documents table even if there aren't any documents?